lib/metalsmith.js

```diff
@@ -46,16 +46,16 @@ module.exports = metalsmith(__dirname) // __dirname defined by node.js: name of
     source: '../' + paths.govukfrontend + 'icons/',
     destination: 'icons'
   }))
-  
+
   // concatenate javascript files
   .use(concat({
-    files: ['javascripts/vendor/modernizr.js', 'javascripts/main.js'],
-    output: 'javascripts/application.js',
+    files: ['javascripts/vendor/modernizr.js', 'javascripts/components/mobile-navigation.js', 'javascripts/main.js'],
```

Comment:
Because Modernizr is included separately in the head we shouldn't _also_ bundle it into `application.js`.

But jQuery _should_ be in here, ideally.

Reply:
My bad @36degrees. Fixed.